Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -249,14 +249,24 @@
   categories:
     - UplinkUtility
 
-#TODO: Increase the price of this to 4-5/remove it when we get encrpytion keys
+#TODO: Increase the price to 4 when flashbang prof.
 - type: listing
   id: UplinkHeadset
-  name: Syndicate Over-ear Headset
-  description: A headset that allows you to listen in on departmental channels, or contact other traitors.
+  name: Syndicate Headset - Over-ear Headset
+  description: A headset that allows you to listen in on departmental channels, or contact our other agents.
   productEntity: ClothingHeadsetAltSyndicate
   cost:
-    Telecrystal: 2
+    Telecrystal: 3
+  categories:
+  - UplinkUtility
+
+- type: listing
+  id: UplinkHeadsetEncryption Key
+  name: Syndicate Headset - Encryption Key
+  description: A encryption key to access the secret frequency of our special agents. Just insert it in any headset you use. And no one will know that you have a special channel with friends... or rivals.
+  productEntity: EncryptionKeySyndie
+  cost:
+    Telecrystal: 1
```

Comment:
Why are you adding the encryption key to the uplink? Not only is 1 TC too cheap (2 TC would be a better price), but it conflicts with the headset. Remove either the headset or the key, or make the headset able to listen to all radio channels with an unremovable key.

Reply:
Seems nice idea